node/src/components/linear_chain_sync.rs

```diff
@@ -162,9 +162,8 @@ pub(crate) struct LinearChainSync<I> {
 }
 
 impl<I: Clone + PartialEq + 'static> LinearChainSync<I> {
-    #[allow(unused)]
     pub fn new<REv: ReactorEventT<I>>(
-        effect_builder: EffectBuilder<REv>,
+        _effect_builder: EffectBuilder<REv>,
```

Comment:
Why do we pass that parameter at all? Are we intending to use it soon? (@goral09?)